requirements/prod-requirements.in

```diff
@@ -1,4 +1,5 @@
 -r requirements.in
+-r sso-requirements.in
```

Comment:
@millerdev my plan for separating `sso-requirements` (as it's an optional app) is to allow separate environment-specific requirements files in `<env>/meta.yml`. I've done this for staging here: https://github.com/dimagi/commcare-cloud/pull/4433

If this looks good, I will remove `-r sso-requirements.in` here so that this PR can be safely merged into master (after a final ✅ ) without affecting deploy, a changelog announcement, or requiring a run of `deploy-stack --tags=apt-get` on all environments in order to install `libxmlsec1-dev` and related `apt` requirements for `python3-saml`.

Reply:
:+1: Seems like a reasonable plan.

Reply:
One question: I assume an error will occur if `ENABLE_SINGLE_SIGN_ON = True` and these requirements are not installed. Would it be possible to make that error message clearly point to resolution steps (special requirements must be installed)?